backend/src/main/java/com/yohoard/notes/DevController.java

```diff
@@ -0,0 +1,59 @@
+package com.yohoard.notes;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Controller
+@RequestMapping("/dev")
```

Comment:
Te fixtury ogólnie nie powinny być w appce. Bo to dane, które chcesz mieć załadowane do bazy przy starcie aplikacji. Niech narazie tak zostanie, ale nie przywiązuj się za mocno do tego kodu, bo to na skrypty init dla mongo trzeba będzie zamienić :P